pandas/tests/groupby/test_groupby.py

```diff
@@ -2012,3 +2012,48 @@ def test_groups_repr_truncates(max_seq_items, expected):
 
         result = df.groupby(np.array(df.a)).groups.__repr__()
         assert result == expected
+
+
+def test_sorted_missing_category_values():
```

Comment:
can you move this test to pandas/tests/groupby/test_categorical.py

Reply:
No problem, will do